### PR TITLE
Hyderabad Data

### PR DESCRIPTION
**Pull request by ank1traj**
Issue number 7

### CODE REVIEW

**Comment by pydevsg** [comment]
Hi, @ank1traj great work indeed .
Please share a screenshot running it on your local server.
Smae goes for other PRs too.

**Comment by ank1traj** [comment]
hyderabad

File: src/Components/Dropdown.js [hunk 1]
```diff
@@ -1,114 +1,133 @@
...
+                href="https://vast-shore-74260.herokuapp.com/banks?city=MUMBAI"
+              >
+                MUMBAI
+              </a>
+              {/*<button onClick={this.fetchData()}>{this.state.ct[4]}</button>*/}
+            </li>
+            < li >
+              <a
+            className = "active"
+            href = "https://vast-shore-74260.herokuapp.com/banks?city=MUMBAI" >
```
Comment (pydevsg) [inline src/Components/Dropdown.js:90]: Please add other cities here, it seems you have written Mumbai thrice which should be written once.

File: src/Components/Dropdown.js [hunk 2]
```diff
@@ -1,114 +1,133 @@
...
+              MUMBAI </a> {/*<button onClick={this.fetchData()}>{this.state.ct[4]}</button>*/ }
+                </li>
+              <li>
+                <a
+              className = "active"
+              href = "https://vast-shore-74260.herokuapp.com/banks?city=MUMBAI" >
```
Comment (pydevsg) [inline src/Components/Dropdown.js:96]: The name of cities should be changed here.

File: src/Components/Dropdown.js [hunk 3]
```diff
@@ -1,114 +1,133 @@
...
+                MUMBAI </a> {
+                  /*<button onClick={this.fetchData()}>{this.state.ct[4]}</button>*/ } </li>
+                < li >
+                  <a
+                className = "active"
+                href = "https://vast-shore-74260.herokuapp.com/banks?city=MUMBAI" >
```
Comment (pydevsg) [inline src/Components/Dropdown.js:102]: Here too , i.e change the API link as well as city names.

File: src/Components/Dropdown.js [hunk 4]
```diff
@@ -1,114 +1,133 @@
...
+                  MUMBAI 
+                  </a> {
+                    /*<button onClick={this.fetchData()}>{this.state.ct[4]}</button>*/ } 
+                  < /li>
+                  <li>
+              <a
+                className="active"
+                href="https://vast-shore-74260.herokuapp.com/banks?city=MUMBAI"
```
Comment (pydevsg) [inline src/Components/Dropdown.js:110]: Same goes for here too.

File: src/Components/Dropdown.js [hunk 5]
```diff
@@ -1,114 +1,133 @@
...
+              </a>
+              {/*<button onClick={this.fetchData()}>{this.state.ct[4]}</button>*/}
+            </li>
+            < li >
+              <
+              a
+            className = "active"
+            href = "https://vast-shore-74260.herokuapp.com/banks?city=MUMBAI" >
+              MUMBAI <
+              /a> {
```
Comment (pydevsg) [inline src/Components/Dropdown.js:122]: and here

File: src/Components/Table.js [hunk 6]
```diff
@@ -36,9 +37,9 @@ class Table extends React.Component {
   }
   //Data update
   componentDidMount() {
-    let ct = ["BANGALORE", "CHENNAI", "DELHI", "KOLKATA", "MUMBAI", "PATNA", "INDORE"];
+    let ct = ["BANGALORE", "CHENNAI", "DELHI", "KOLKATA", "MUMBAI", "PATNA", "INDORE", "PUNE", "HYDERABAD"];
     let url = "https://vast-shore-74260.herokuapp.com/banks?city=";
-      for (let i = 0; i < 7; i++) {
+      for (let i = 0; i < 5; i++) {
```
Comment (pydevsg) [inline src/Components/Table.js:42]: It should be i=0;i<9;

**Review by pydevsg — CHANGES_REQUESTED** [review]
@ank1traj please make all the changes as suggested by me.

**Comment by ank1traj** [comment]
Done. Kindly review once again.

**Pull request closed without merging**